application/common/mail/reset/on-behalf.php

```diff
@@ -18,17 +18,24 @@
 </p>
 <p>
     <?= yHtml::encode($displayName) ?> recently requested a password change for their
-    <?= yHtml::encode($idpDisplayName) ?> Identity account. If this was you, please use the link below
-    to reset your password. If it's not you but you do know them, you may provide the link
-    for them to use, but please contact them directly to ensure that you are only providing the
-    link to them and not to someone else. This link is valid
-    until <?= yHtml::encode($expireTime) ?>.
+    <?= yHtml::encode($idpDisplayName) ?> Identity account. 
+</p>
+<p>
+    If this was you, please use the link below to reset your password. 
+</p>
+<p>
+    If it's not you but you do know them, you may have been sent this link because they requested it sent to you - their recovery contact. You may provide the link for them to use, but <i>please contact them directly</i> to ensure that you are only providing the link to them and not to someone else. 
```

Comment:
If you don't mind, please break this up into three lines so it's easier to see on the screen (I don't think that affects the email contents)

Reply:
updated...i think